tests/utils/test_env.py

```diff
@@ -502,7 +498,7 @@ def test_deactivate_activated(
         del os.environ["VIRTUAL_ENV"]
 
     venv_name = manager.generate_env_name("simple-project", str(poetry.file.parent))
-    version = Version.parse(".".join(str(c) for c in sys.version_info[:3]))
+    version = Version.from_parts(*sys.version_info[:3])
```

Comment:
Awesome change! This pattern is really common and I'd like to see us replace it more generally.